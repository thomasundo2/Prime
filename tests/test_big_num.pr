int main()
{
    lint l1;
    lint l2;
    l1 = 100000000000000000l;
    l2 = 200000000000000000l;

    printl(l1);
    printl(l1 + l2);
    printl(l1*l2);
    printl(l1^l1@(l1*3l));

}
