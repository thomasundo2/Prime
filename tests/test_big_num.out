100000000000000000
300000000000000000
20000000000000000000000000000000000
100000000000000000
